README.md

```diff
@@ -1,35 +1,37 @@
 # Spinnaker-Gardening #CommunityHack
 
-Move the needle on continuous delivery projects at Spinnaker Gardening Days #communityHack, sponsored by SalesForce! Join user companies to learn and let your skills shine at this collaborative hackathon. We’ll address open-source feature requirements and extend the ecosystem.
+Move the needle on continuous delivery projects at Spinnaker Gardening Days #communityHack, sponsored by SalesForce! Join other Spinnaker users and companies to learn and let your skills shine at this collaborative hackathon. We’ll address open-source feature requests and extend the ecosystem.
 
 ## What is this?
-This is documentation for Spinnaker Gardening Days #communityHack, a series of open-source Spinnaker hackathons open to anyone in the community, including newcomers. The goal of this doc is to set expectations and help you get started.
+This is documentation for the Spinnaker Gardening Days #communityHack, a series of open-source Spinnaker hackathons open to anyone in the community, including newcomers. The goal of this doc is help you get started.
 
 ## When is the event?
 The first event will take place online on April 9 & 10, 2020. [Click here](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111) to register. Tickets are available for the hackathon, 
 
 ## How should I prepare to participate in Spinnaker Gardening?
-1. Register for the event [here in Eventbrite](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111)
+1. Register for the event [on Eventbrite](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111)
 2. [Join Spinnaker Slack](join.spinnaker.io) and join the following channels:
-    * **#gardening-general**   Make general, public conversation. Please use threads for (welcome!) jokes and side convos
-    * **#gardening-ineedhelp**   Request help here from experts during the hackathon, or ask setup questions before it starts.
-3. Ask for your organization to "sponsor" your time at the event. [Use this flyer](https://s.armory.io/v1ur0KQQ) to highlight the benefits of participating.
-4. Visit the [Project Ideas Wiki](https://github.com/spinnaker-hackathon/gardening/wiki/Project-Ideas) in the gardening wiki tab. 
+    * **[#gardening-general](https://spinnakerteam.slack.com/archives/CV4A90DPF)**   Make general, public conversation. Please use threads for (welcome!) jokes and side conversations.
+    * **[#gardening-ineedhelp](https://spinnakerteam.slack.com/archives/CURFZGL2E)**   Request help from experts during the hackathon, or ask setup questions before it starts.
+3. Ask your organization to "sponsor" your time at the event. [Use this flyer](https://s.armory.io/v1ur0KQQ) to highlight the benefits of participating.
```

Comment:
@dnilasor you might want to elaborate on what you mean by "sponsor" because 1) scare quotes are scary and 2) people are automatically going to associate that with $$$$

Reply:
@brian-armory excellent thought will address!